test/browser/features/Welsh_DVA_Driving_Licence.feature

```diff
@@ -62,6 +62,14 @@ Feature: DVA Driving licence CRI Error Validations
     Given I can see the postcode field titled Cod post
     Then I can see postcode sentence as Rhowch y cod post yn y cyfeiriad yn adran 8 o'ch trwydded
 
+  @mock-api:dva-ConsentSection @validation-regression @build @staging@Language-regression
```

Comment:
@MathumithaVe The welsh tests don't need the build, staging or validation regressions tags